lib/elixir/lib/system.ex

```diff
@@ -5,43 +5,64 @@ defmodule System do
   with the VM or the host system.
   """
 
-  defp strip_re(iodata, pattern) do
-    :re.replace(iodata, pattern, "", [return: :binary])
+  @base_dir           Path.join([__DIR__, "../../.."])
```

Comment:
It would actually nice to not use `Path.join` function, the same way we didn't use it before, because using Path can put us into bootstrapping issues. Sure, it works now, but it may not work in a month if we decide to improve Path.